instance/conn_name.go

```diff
@@ -74,3 +75,9 @@ func ParseConnName(cn string) (ConnName, error) {
 	}
 	return c, nil
 }
+
+// ConnectionNameResolver resolves  This allows an application to replace the default
+// DNSInstanceConnectionNameResolver with a custom implementation.
```

Comment:
Maybe just "replace the default instance connection name resolution..."

Reply:
fixed.